alf/algorithms/mi_estimator.py

```diff
@@ -11,22 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
mi_estimator is already included in another PR

Reply:
will wait for PR 513 to merge first.